### ISSUE TITLE
Linux wheels

### ISSUE THREAD

**Issue by bencardoen** [ZhuangLab/storm-analysis#35]
Hi,

I'm trying to install your project, but have issues with compiling it from source.
Do you have linux wheels available? I can see the windows ones, but I need to install this on a cluster running Linux.

Thanks!

Ben

**Comment by HazenBabcock** [comment]
Apologies, currently there are no linux wheels. I could make a 64bit wheel for 2.7 or 3.6 and share it with you. The C libraries might not be optimized for your computer though. Not sure how much difference that might make. Git head? Or do you want a particular version?

Can you describe the issue you are having? Maybe paste the error message here? A linux build should just work so I would like to see what the issue might be.

**Comment by bencardoen** [comment]
Thanks for getting back to me so quickly,

I'm trying to build your project on a cluster where GCC can't find lapack (long story). I'm trying to work around that, but I saw in the scons config file that you had standalone versions of those libraries so I tried to configure the scons file to build them (as it does for windows I think). I don't have that much experience with scons and my attempts to force it to build standalone on Linux failed.

In any case, if you could provide a wheel for Linux 64bit, Python 3.6 that would help me a lot. 

In terms of version, I'll rely on your judgement, I prefer git head unless it's not stable.

As a side note, I'm really impressed with the quality of your code, docs & tests, I've been testing a lot of superresolution reconstruction methods and yours stands out by far. 

Thanks!

Ben

**Comment by HazenBabcock** [comment]
I shared a linux wheel on Google Drive:
[linux wheel](https://drive.google.com/drive/folders/1SAmKaaKYM1QJtta-ySpe-QUy4gYmGMiq?usp=sharing)

This is git head (bc4733c3482018c06c354e63fa508dc61a3d3956). It should be stable. The biggest change relative to 2.0 is that I removed the 2x oversampling that I was using in cubic spline fitting since this did not improve fitting and just made Spliner slower. If you were using Spliner you will have to remeasure your PSF splines.

I'm not sure this going to work for you without a Lapack library somewhere on your system. I'm pretty sure the dependency still remains. The build that I did did not compile my Lapack library into the storm-analysis C libraries (the libraries are dynamic, not static).

Yeah, the included lapack library is for Windows so you definitely would not be able to use it on Linux.
